build.sbt

```diff
@@ -23,6 +23,8 @@ ThisBuild / scalafixAll / skip := tlIsScala3.value
 ThisBuild / ScalafixConfig / skip := tlIsScala3.value
 ThisBuild / circeRootOfCodeCoverage := Some("rootJVM")
 
+ThisBuild / evictionErrorLevel := Level.Info
```

Comment:
I dont think we should do this.
This will change the error level for all dependencies.
I would rather that we have explicit overrides for those we know are safe.